<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="W-7FM3M13" failures="3" tests="6" name="Platform Sign Up" time="399.816" errors="0" timestamp="2020-11-12T21:57:23 IST">
  <testcase classname="com.test.testcases.PlatformSignUpTest" name="signUpConversationalAI" time="67.385">
    <skipped/>
  </testcase> <!-- signUpConversationalAI -->
  <testcase classname="com.test.testcases.PlatformSignUpTest" name="signUpConversationalAI" time="69.941">
    <failure type="java.lang.AssertionError" message="Success Message does not exist">
      <![CDATA[java.lang.AssertionError: Success Message does not exist
at com.utils.GenericUtils.checkFieldExist(GenericUtils.java:192)
at com.pages.SignUpCAIPage.verifySuccessMessage(SignUpCAIPage.java:141)
at com.test.testcases.PlatformSignUpTest.signUpConversationalAI(PlatformSignUpTest.java:62)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- signUpConversationalAI -->
  <testcase classname="com.test.testcases.PlatformSignUpTest" name="signUpConversationalAI" time="68.442">
    <skipped/>
  </testcase> <!-- signUpConversationalAI -->
  <testcase classname="com.test.testcases.PlatformSignUpTest" name="signUpConversationalAI" time="68.484">
    <failure type="java.lang.AssertionError" message="Success Message does not exist">
      <![CDATA[java.lang.AssertionError: Success Message does not exist
at com.utils.GenericUtils.checkFieldExist(GenericUtils.java:192)
at com.pages.SignUpCAIPage.verifySuccessMessage(SignUpCAIPage.java:141)
at com.test.testcases.PlatformSignUpTest.signUpConversationalAI(PlatformSignUpTest.java:62)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- signUpConversationalAI -->
  <testcase classname="com.test.testcases.PlatformSignUpTest" name="signUpConversationalAI" time="63.343">
    <skipped/>
  </testcase> <!-- signUpConversationalAI -->
  <testcase classname="com.test.testcases.PlatformSignUpTest" name="signUpConversationalAI" time="59.471">
    <failure type="java.lang.AssertionError" message="Success Message does not exist">
      <![CDATA[java.lang.AssertionError: Success Message does not exist
at com.utils.GenericUtils.checkFieldExist(GenericUtils.java:192)
at com.pages.SignUpCAIPage.verifySuccessMessage(SignUpCAIPage.java:141)
at com.test.testcases.PlatformSignUpTest.signUpConversationalAI(PlatformSignUpTest.java:62)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- signUpConversationalAI -->
</testsuite> <!-- Platform Sign Up -->
